openbas-api/src/main/java/io/openbas/rest/inject/output/InjectOutput.java

```diff
@@ -39,7 +42,7 @@ public class InjectOutput {
   private Long dependsDuration;
 
   @JsonProperty("inject_depends_on")
-  private String dependsOn;
+  private Map<String, String> dependsOn;
```

Comment:
I wanted to have only the minimal information needed on the frontend side. In retrospect, that added complexity and bugs so it probably would have been better to keep using the bean. 
I'm not too happy about pushing to prod something we already know will need some refactor but to avoid pushing back the release date for this feature, would it be okay to do it in #1615 ?

Reply:
Let's try this way.
I don't know where we need to write it to be sure we will male the chanegs.